scripts/rollup/build.js

```diff
@@ -454,6 +517,7 @@ function getPlugins(
       }),
     // License and haste headers, top-level `if` blocks.
     {
+      name: 'license-and-headers',
       renderChunk(source) {
         return Wrappers.wrapBundle(
           source,
```

Comment:
See my notes/exploration/proposal on a previous PR for how this `renderChunk` and the related `Wrappers.wrapBundle` code could make use of `magic-string` for sourcemap support:

- https://github.com/facebook/react/pull/21946#discussion_r917339710 (and following comments)